LayoutTests/imported/w3c/web-platform-tests/css/css-masking/parsing/clip-path-computed-expected.txt

```diff
@@ -8,10 +8,12 @@ PASS Property clip-path value 'xywh(0px 1% 2px 0.5em round 0)'
 PASS Property clip-path value 'xywh(0px 1% 2px 3% round 0 1px)'
 PASS Property clip-path value 'xywh(0px 1% 2px 3% round 0px 1px 2em)'
 PASS Property clip-path value 'xywh(0px 1% 2px 3% round 0px 1px 2% 3px)'
+FAIL Property clip-path value 'xywh(calc(0px) calc(1px + 1%) calc(2px + 2%) calc(3px + 3%))' assert_equals: expected "inset(calc(1% + 1px) calc(98% - 2px) calc(96% - 4px) 0px)" but got "inset(calc(1% + 1px) calc(100% - (2% + 2px)) calc(100% - (4% + 4px)) 0px)"
```

Comment:
Why not solve the “100% - (2% + 2px)” problem too?

Reply:
It is better solved separately as it is not clear what is the right path for fixing it.